### DISCUSSION TITLE
Disable Server-Side ResultSet Data Filter Button

### DISCUSSION THREAD

**Discussion by imthenalls** [dbeaver/dbeaver#12366] [Q&A]
Is there any way to disable the server-side data filtering that the filter button does by default?
I'm talking about the one that's in the result set.
There don't seem to be any settings in preferences to disable it.

**Comment by LonwoLonwo** [comment]
Hello @imthenalls 
What database do you use? 
Can you provide some examples?

**Comment by LonwoLonwo** [comment]
Probably, this is the setting:

![2023-12-26 18_15_41-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/76d40866-139b-4e54-9174-309406368553)
